pkg/preference/preference.go

```diff
@@ -112,18 +112,18 @@ func NewPreference() Preference {
 // NewPreferenceInfo gets the PreferenceInfo from preference file and creates the preference file in case it's
 // not present
 func NewPreferenceInfo() (*PreferenceInfo, error) {
-	configFile, err := getPreferenceFile()
-	glog.V(4).Infof("The preference file is %+v", configFile)
+	preferenceFile, err := getPreferenceFile()
+	glog.V(4).Infof("The preference file is %+v", preferenceFile)
```

Comment:
Thanks for fixing this! No more `configFile` vs. `preferenceFile` confusion. :smile: